UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common.UnitTests/Logging/EventIdExtensionsTest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NUnit.Framework;
+using UKHO.ExchangeSetService.Common.Logging;
+
+namespace UKHO.ExchangeSetService.Common.UnitTests.Logging
+{
+    [TestFixture()]
+    public class EventIdExtensionsTest
+    {
+        [Test()]
+        public void WhenExtensionSetEventId_ThenValidateItsISEqualToActualValue()
```

Comment:
Change this typo to public void WhenExtensionSetEventId_ThenValidateItIsEqualToActualValue()

Also remove () from Test and TestFixture

Reply:
made changes accordingly.